File: app/src/main/java/com/projects/asgrebennikov/repetitor/MainActivity.java
package com.projects.asgrebennikov.repetitor;

import android.content.Intent;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import com.google.android.gms.ads.MobileAds;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button practiceButton = (Button) findViewById(R.id.startPracticeButton);
        practiceButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public synchronized void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, BookSelectionActivity.class);
                startActivity(intent);
            }
        });

        Button learningButton = (Button) findViewById(R.id.startLearningButton);
        learningButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public synchronized void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, LearningWordsActivity.class);
                startActivity(intent);
            }
        });

        Button feedbackButton = (Button) findViewById(R.id.writeFeedbackButton);
        feedbackButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public synchronized void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, GetFeedbackActivity.class);
                startActivity(intent);
            }
        });

        //MobileAds.initialize(this, "ca-app-pub-2381885173378825~5894393156");
    }
}

File: app/src/main/java/com/projects/asgrebennikov/repetitor/MainActivity.kt
package com.projects.asgrebennikov.repetitor

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val practiceButton = findViewById<View>(R.id.startPracticeButton) as Button
        practiceButton.setOnClickListener {
            val intent = Intent(this@MainActivity, BookSelectionActivity::class.java)
            startActivity(intent)
        }

        val learningButton = findViewById<View>(R.id.startLearningButton) as Button
        learningButton.setOnClickListener {
            val intent = Intent(this@MainActivity, LearningWordsActivity::class.java)
            startActivity(intent)
        }
    }
}
